feature/extrasA/src/main/resources/modules/system/layers/fuse/org/apache/camel/component/jcr/main/module.xml

```diff
@@ -4,9 +4,9 @@
     <artifact name="${org.apache.camel:camel-jcr}" />
   </resources>
   <dependencies>
+    <module name="com.rometools.rome" slot="1.0" />
```

Comment:
Is the slot change intended for this component?

Reply:
The intention is to have two rome slots: one with 1.0 to support jcr and rss components (as before) and main holding rome 1.5 to support tika. It works quite well like this because rome 1.0 has different groupId than rome 1.5 (rome vs. com.rometools.rome).